carbon-design-system/carbon-platform
# feat(web-app): add express proxy server to log request

Closes #426 

- [x] cleanup web-app now unuseful code
- [x] figure out port thing, how does this work?

**Testing/Reviewing**
 
- `npm install`
- run with `npm run dev`, make sure app runs as expected both on `http://localhost:8080` and on `http://localhost:3001`
- run with `npm run dev:secure`, make sure app runs as expected on `https://localhost:8443` (might need to set auth variables -CARBON_IBM_ISV_CLIENT_ID, CARBON_IBM_ISV_CLIENT_SECRET, PASSPORT_STRATEGY_NAME- to be able to get through the login)
